cirq/contrib/contrib-requirements.txt

```diff
@@ -1,4 +1,4 @@
 # Runtime requirements for contrib.
 
 pylatex==1.3.*
-tensorflow
+tf-nightly
```

Comment:
Tensorflow for Python3.7 is currently only supported in the nightly build. I will add a comment above this line stating this case in the next commit.